cypress/integration/HomeworkTests/Calculator.spec.js

```diff
@@ -0,0 +1,124 @@
+import Calculator from "../../page_objects/homepage/calculator.js";
+/// <reference types="cypress" />
+
+var calculator = new Calculator()
+
+var firstInput = 2.5;
+var secondInput =2;
+
+var sumInputs = firstInput + secondInput;
+var subInputs = firstInput - secondInput;
+var multInputs = firstInput * secondInput;
+var divInputs = firstInput / secondInput;
+var concatInputs = firstInput.toString() + secondInput.toString();
+
+
+var buildVers = ['Prototype', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+
+beforeEach("Executes before each test",()=>{
+    cy.visit("https://testsheepnz.github.io/BasicCalculator")
+})
+
+it("1. Loads the TestSheepNz Calculator page", ()=>{
+   cy.contains('Calculate')
+});
+
+
+buildVers.forEach((buildVer) => {
+
+it("1.2 Checks if two input forms exists", ()=>{
```

Comment:
Looks like indentation is a bit off.  These tests are under forEach cycle, so should be moved right.